parser/js/test/unit/decoder/covtDecoder.spec.ts

```diff
@@ -1,13 +1,26 @@
 import * as fs from "fs";
 import { MVTLayer, parseMvtTile } from "../../../src/mvtUtils";
 import { CovtDecoder } from "../../../src/decoder/covtDecoder";
+import mvtFixtures from "@mapbox/mvt-fixtures";
 import * as Path from "path";
 import { toBeDeepCloseTo, toMatchCloseTo } from "jest-matcher-deep-close-to";
 expect.extend({ toBeDeepCloseTo, toMatchCloseTo });
 
 const tilesDir = "./data";
 
 describe("CovtDecoder", () => {
+    it("should decode a simple valid tile", () => {
+        const tiles = getMvtFixtureTiles("017"); // Small valid MVT fixture
```

Comment:
You could also do `mvtFixtures.get('017')` (https://github.com/mapbox/mvt-fixtures/blob/main/API.md#get)